http/src/test/groovy/io/micronaut/http/uri/UriTemplateSpec.groovy

```diff
@@ -33,24 +33,32 @@ class UriTemplateSpec extends Specification {
-        ''             | '/hello/{name}'      | [name: 'Fred']                          | '/hello/Fred'
-        '/test/'       | '/hello/{name}'      | [name: 'Fred']                          | '/test/hello/Fred'
-        '{var}'        | '{var2}'             | [var: 'foo', var2: 'bar']               | 'foo/bar'
-        '/book{/id}'   | '/author{/authorId}' | [id: 'foo', authorId: 'bar']            | '/book/foo/author/bar'
-        '{var}/'       | '{var2}'             | [var: 'foo', var2: 'bar']               | 'foo/bar'
-        '{var}'        | '/{var2}'            | [var: 'foo', var2: 'bar']               | 'foo/bar'
-        '{var}{?q}'    | '/{var2}'            | [var: 'foo', var2: 'bar', q: 'test']    | 'foo/bar?q=test'
-        '{var}{?q}'    | '{var2}'             | [var: 'foo', var2: 'bar', q: 'test']    | 'foo/bar?q=test'
-        '{var}{#hash}' | '{var2}'             | [var: 'foo', var2: 'bar', hash: 'test'] | 'foo/bar#test'
-        '/'            | '{?req*}'            | [req: [var: 'foo', var2: null]]         | '/?var=foo'
+        template       | nested               | arguments                                 | result
+        '/city'        | 'country/{name}'     | [name: 'Fred']                            | '/city/country/Fred'
+        '/city/'       | 'country/{name}'     | [name: 'Fred']                            | '/city/country/Fred'
+        '/city/'       | '/country/{name}'    | [name: 'Fred']                            | '/city/country/Fred'
+        '/city'        | '/country/{name}'    | [name: 'Fred']                            | '/city/country/Fred'
+        '/poetry'      | '/{?max}'            | [max: '10']                               | '/poetry?max=10'
+        '/poetry'      | '{?max}'             | [max: '10']                               | '/poetry?max=10'
+        '/'            | '/hello/{name}'      | [name: 'Fred']                            | '/hello/Fred'
+        ''             | '/hello/{name}'      | [name: 'Fred']                            | '/hello/Fred'
+        '/test/'       | '/hello/{name}'      | [name: 'Fred']                            | '/test/hello/Fred'
+        '{var}'        | '{var2}'             | [var: 'foo', var2: 'bar']                 | 'foo/bar'
+        '/book{/id}'   | '/author{/authorId}' | [id: 'foo', authorId: 'bar']              | '/book/foo/author/bar'
+        '{var}/'       | '{var2}'             | [var: 'foo', var2: 'bar']                 | 'foo/bar'
+        '{var}'        | '/{var2}'            | [var: 'foo', var2: 'bar']                 | 'foo/bar'
+        '{var}{?q}'    | '/{var2}'            | [var: 'foo', var2: 'bar', q: 'test']      | 'foo/bar?q=test'
+        '{var}{?q}'    | '{var2}'             | [var: 'foo', var2: 'bar', q: 'test']      | 'foo/bar?q=test'
+        '{var}{#hash}' | '{var2}'             | [var: 'foo', var2: 'bar', hash: 'test']   | 'foo/bar#test'
```

Comment:
Can you please revert the spacing changes here?

Reply:
Reverted, I had Intellij to ignore whitespaces so this one slipped through.